### DISCUSSION TITLE
[@types/diff] does not properly export Diff.reversePatch

### DISCUSSION THREAD

**Discussion by wearrrrr** [DefinitelyTyped/DefinitelyTyped#69180] [Issues with a @types package]
I assume this is just an issue with the function being added somewhat recently, but as it stands DT's implementation of @types/diff does not have support for Diff.reversePatch(structuredPatch), which as the parameter indicates, passes in a structured patch, this isn't critical but it would be nice to see support added :slightly_smiling_face: 

Commit where it was added: https://github.com/kpdecker/jsdiff/commit/83653673cbe22e1688985ccd3c468b418f48d6bb

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment]
PR: #69183 69183
